hl7/4.2/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,174 @@
+= HL7 EDI Connector 4.2 - Mule 4
+:page-aliases: connectors::hl7/hl7-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+Anypoint Connector HL7 EDI (HL7 EDI Connector) is a collection of features that facilitate integration with healthcare systems by providing the tools needed to easily create, read and transform HL7 version v2.x messages within Mule. Health Level Seven International (HL7) is a standards development organization that defines a set of standards for the transport and communication of healthcare information.
+
+For compatibility information and fixed issues, see the HL7 EDI Connector Release Notes.
+
+Standard schema definitions are included in HL7 EDI Connector for
+HL7 versions 2.1, 2.2, 2.3.1, 2.3, 2.4, 2.5, 2.5.1, 2.6, 2.7,
+2.7.1, 2.8, 2.8.1. The schemas are provided in two forms:
+
+* Standard HL7: using required or optional status data types,
+field lengths, and repetition counts (as listed) for each HL7 version. +
+
+* Lax HL7: all segments and value items are optional. These
+low-level data types are treated as simple strings. Field lengths and repetition counts are not enforced.
+
+Both forms of schemas are available in the connector `classpath`. Standard HL7
+schemas use the path: `/hl7/v2_5_1/ADT_A01.esl` +
+Lax schemas use paths: `/hl7lax/v2_5_1/ADT_A01.esl`
+
+The schema definitions you need for xref:hl7-schemas.adoc[HL7 Supplied Schemas] are included.
+
+For complete information about schemas, see xref:x12-edi-connector::x12-edi-schema-language-reference.adoc[EDI Schema Language Reference].
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+* HL7 EDI Connector
+
+To use HL7 EDI Connector, you must have:
+
+* Purchased a MuleSoft license for Anypoint B2B
+* Anypoint Studio 7.0 or later
+
+=== Understanding the EDI Schema Language
+
+The HL7 EDI Connector uses a YAML format called EDI Schema Language (ESL) to represent EDI schemas.
+Basic ESLs define the structure of Electronic Data Interchange (EDI) messages in terms of:
+
+* structures (message structures, in HL7 terminology)
+
+* groups
+
+* segments, composites, and elements
+
+You can customize the HL7 schema definitions to suit your data by
+copying and editing the schema definitions directly or by using a
+console tool to generate a simplified schema based on one or more
+sample documents. It is also possible to use an overlay schema for
```

Comment:
```suggestion
sample documents. Alternatively, you can use an overlay schema for
```
GLOBALLY reword any sentence that begins with "There is," "There are," "It is," and so on: false sentence subjects.